app/components/forms/wizard/other-details-step.js

```diff
@@ -0,0 +1,166 @@
+      });
+    }
+
+    return validationRules;
+  }),
+
+
+  showDraftButton: computed('data.event.state', function() {
+    return this.data.event.state !== 'published';
+  }),
+
+  hasCodeOfConduct: computed('data.event.codeOfConduct', function() {
+    return !!this.data.event.codeOfConduct;
+  }),
+
+  didInsertElement() {
+    if (!this.isCreate && this.data.event.copyright && !this.data.event.copyright.content) {
+      this.set('data.event.copyright', this.store.createRecord('event-copyright'));
+    }
+  },
+
+  // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
+  getValidationRules() {
+    $.fn.form.settings.rules.checkMaxMinPrice = () => {
+      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+    };
+    $.fn.form.settings.rules.checkMaxMinOrder = () => {
+      return $('.ui.form').form('get value', 'ticket_min_order') <= $('.ui.form').form('get value', 'ticket_max_order');
+    };
+
+    const validationRules = {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        timezone: {
+          identifier : 'timezone',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Choose a timezone for your event')
+            }
+          ]
+        },
+        startTime: {
+          identifier : 'start_time',
+          depends    : 'start_date',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please give a start time')
+            }
+          ]
+        },
+        endTime: {
+          identifier : 'end_time',
+          depends    : 'end_date',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please give an end time')
+            }
+          ]
+        },
+        externalEventIdentifier: {
+          identifier : 'external_event_url',
+          optional   : true,
+          rules      : [
+            {
+              type   : 'regExp',
+              value  : protocolLessValidUrlPattern,
+              prompt : this.l10n.t('Please enter a valid url')
+            }
+          ]
+        },
+      }
+    };
+    // Merging the predetermined rules with the rules for social links.
+    validationRules.fields = merge(validationRules.fields, this.socialLinksValidationRules);
+    return validationRules;
+  },
+
+  actions: {
+    async updateCopyright(name) {
+      const { event } = this.data;
+      const copyright = await this.getOrCreate(event, 'copyright', 'event-copyright');
+      const license = find(licenses, { name });
+      copyright.setProperties({
+        licence    : name,
+        logoUrl    : license.logoUrl,
+        licenceUrl : license.link
+      });
+    },
+     saveDraft() {
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces but found 5.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6089057)